core/src/commonMain/kotlin/com/ustadmobile/core/view/ContentEntryEdit2View.kt

```diff
@@ -22,6 +22,12 @@ interface ContentEntryEdit2View: UstadEditView<ContentEntryWithLanguage>{
 
     var entryMetaData: ImportedContentEntryMetaData?
 
+    var entryCompressed: Boolean
+
+    var videoDimensions: Pair<Int, Int>
```

Comment:
This is really only a getter. It should be a val with only a getter (no field needs to be initialized).

Reply:
done